### ISSUE TITLE
Erroneous attempts to set peer data 

### ISSUE THREAD

**Issue by mikera** [Convex-Dev/convex#429]
Noticed on Discord:

```
moneyisdebt — Today at 06:26
I just tried starting a new peer - if you look at block 633 in testnet, there's a sequence number error? Is that common, or am I doing something wrong?
Mike — Today at 07:40
sequence number error is due to a transaction having the wrong sequence number (it's basically a client error).
I see (set-peer-data 0x7350e9ae11939b93229abb33911c81fe38c5837060489994b33c8dc3d29272da {:url "0.0.0.0:8200"}) is that using the latest code base? 
Basically the peer is acting as a client and trying to announce it's public URL (but you don't have one so it's not really useful....)
```

**Comment by mikera** [comment]
This has been fixed by recent updates - not observed again recently
